src/librustdoc/clean/mod.rs

```diff
@@ -3147,7 +3147,10 @@ impl<'tcx> Clean<Constant> for ty::Const<'tcx> {
     fn clean(&self, cx: &DocContext<'_>) -> Constant {
         Constant {
             type_: self.ty.clean(cx),
-            expr: format!("{:?}", self.val), // FIXME(const_generics)
+            expr: match self.val {
+                ConstValue::Param(ty::ParamConst { name, .. }) => format!("const {}", name),
+                e => format!("{:?}", e), // FIXME generic consts
```

Comment:
I don't think it's entirely clear what's still to fix.

Reply:
In general, the constant could be an arbitrary expression, in which case we want to pretty-print it rather than debug format it.